src/coreclr/src/pal/src/exception/machexception.cpp

```diff
@@ -480,6 +510,28 @@ BuildExceptionRecord(
                 case EXC_I386_BOUND:
                     exceptionCode = EXCEPTION_ARRAY_BOUNDS_EXCEEDED;
                     break;
+#elif defined(HOST_ARM64)
+                case EXC_ARM_FP_IO:
```

Comment:
Is there no exception for integer div by 0?

Reply:
There is not.  From my recollection, arm64 doesn't have an integer divide by zero instruction. So the divide emulation must handle this.